test_pics/read_before_deleting.md

```diff
@@ -0,0 +1,5 @@
+**Attention**
+
+This folder is for test images generation.  
+
+As github does not allow empty folders, this file should be in place.  
```

Comment:
Could you elaborate why this is necessary? Usually one wants to avoid file operations as part of unit tests, but read a fixed image into memory. The other API should not directly operate on files but only on `impl Read` and `impl Write` types and compare with the reference images.

If needed, one can add file base test programs under `tests` to generate the _reference_ images on certain occassions (breaking version change i.e.).

Reply:
Removed tests with image writing, left two test to read and compare with known png and svg images.